trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/add-to-an-existing-payment-helper.test.ts

```diff
@@ -0,0 +1,93 @@
+import { SelectedFeeRecordsAvailablePaymentDetails, SelectedFeeRecordsAvailablePaymentGroups } from '@ukef/dtfs2-common';
+import { getAvailablePaymentsHeading } from './add-to-an-existing-payment-helper';
+
+describe('add-to-an-existing-payment-helper', () => {
+  describe('getAvailablePaymentsHeading', () => {
+    it('should throw an error when no payment groups are provided', () => {
+      // Arrange
+      const paymentGroups: SelectedFeeRecordsAvailablePaymentGroups = [];
+
+      // Act & Assert
+      expect(() => getAvailablePaymentsHeading(paymentGroups)).toThrow('No payment groups to select from.');
+    });
+
+    it('should return correct heading for a single payment', () => {
```

Comment:
Please can we lose the word 'correct' from all these test titles replacing it with a description of what is specific about the header in these cases?